js/database.js

```diff
@@ -0,0 +1,66 @@
+import { getToday, incrementDay, isSameDate, MILISECOND_HOUR } from "./dateManipulation.js"
+
+if(!localStorage.getItem("eventId")){
```

Comment:
I would suggest not to execute code as soon as file is imported - most developers would not expect it and it could lead to weird errors in larger projects. If there is some initialization required, there are other ways to do it. You could have some init function, you could have class with constructor, etc.

In this case since I suggested having getEventId selector function in a comment below - maybe you could write selector like this:

`localStorage.getItem("eventId") || 0`